homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+  .tips-sub__item-text-top {
+    margin-top: 0;
+  }
+  .item-title{
+    font-size: 42px;
+    line-height: 54px;
+    max-width: 359px;
+    width: 100%;
+  }
+  .info-item {
+    width: 400px;
+    margin-top: 63px;
+  }
+  .border-radius {
+    width: 37px;
+    height: 38px;
+    right: 47px;
+    top: 55px;
+    border: 2px solid #22BF95;
+  }
+  .mobile_img {
+    display: none;
+  }
+  .dekstop_img {
+    display: block;
+  }
+  .border-img {
+    padding: 20px 0 0;
+  }
+  .glossary {
+    padding-top: 160px;
+  }
+  .glossary__img {
+    top: 2px;
+  }
+  .slider__wrapper {
+    margin-top: 50px;
+  }
+  .slider__wrapper .slider__inner {
+    padding: 60px 0;
+    flex-direction: row;
+    justify-content: start;
+  }
+  .slider__wrapper .slider__inner .slider-nav {
+    max-width: 422px;
+    width: 100%;
+    padding: 60px 0 27px;
+  }
+  .slider__wrapper .slider__inner .slider-single {
+    max-width: 770px;
+    width: 100%;
+  }
+  .slider__wrapper .slider__inner .slider-single .slider__title-wrapper {
+    margin-left: 100px;
+  }
+  .slider__wrapper .slider__inner .slider-single .slider__title-wrapper .slider__text-title {
+    font-size: 90px;
+    line-height: 120px;
+  }
+  .slider__wrapper .slider__inner .slider-nav .slick-slide.is-active h3:before {
+    left: 0;
+    height: 1px;
+    width: 105px;
+    background: #22BF95;
+  }
+  .slider__wrapper .slider__inner .slider-nav h3 span {
+    margin-left: 29px;
+    left: 50%;
+  }
+  .slider__wrapper .slider__inner .slider-nav h3:hover::before{
+    content: "";
+    position: absolute;
+    left: 0;
+    height: 1px;
+    width: 105px;
+    background: #22BF95;
+  }
+  .slider__wrapper .slider__inner .slider-single .slider__title-wrapper .slider__text-text {
+    margin-top: 73px;
+    font-size: 24px;
+    line-height: 36px;
+    max-width: 553px;
+    position: relative;
+    padding-left: 102px;
+  }
+  .slider__wrapper .slider__inner .slider-single .slider__title-wrapper .slider__text-text::before{
+    background-color: #22BF95;
+    height: 72px;
+    left: 65px;
+  }
+  .slider__wrapper .slider__inner .prev, .slider__wrapper .slider__inner .next{
+    width: 25px;
+  }
+  .slider__wrapper .slider__inner .prev {
+    top: 54px;
+  }
+  .fqas {
+    padding: 160px 0 100px;
+  }
+  .fqas__link {
+    width: 950px;
+    display: flex;
+    align-items: center;
+  }
+  .fqas-btn{
+    color: #22BF95;
+  }
+  .fqas__link::before, .fqas__link::after {
+    background-color: #22BF95;
+    width: 22px;
+    top: 60px;
+  }
+  .fqas__link::after {
+    right: -104px;
+  }
+  .fqas-item__num {
+    font-size: 110px;
+    line-height: 130px;
+  }
+  .fqas-item__title {
+    max-width: 100%;
+    width: 100%;
+    font-size: 24px;
+    line-height: 36px;
+  }
+  .answer-title{
+    font-size: 16px;
+    max-width: 670px;
+    width: 100%;
+  }
+  .fqas__link_active {
+    width: 950px;
+  }
+  .footer {
+    padding-top: 77px;
+  }
+  .footer__container {
+    max-width: 1347px;
```

Comment:
Try to avoid such numbers. Try to use just container class

Reply:
fixed a bug